arm-sql/2014-04-01/swagger/databaseSecurityAlertPolicies.json

```diff
@@ -0,0 +1,280 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Database Threat Detection Policy APIs",
+    "description": "Provides create, read and update functionality for Azure SQL database Threat Detection policies.",
+    "version": "2014-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/securityAlertPolicies/{securityAlertPolicyName}": {
+      "get": {
+        "tags": [
+          "DatabaseThreatDetectionPolicies"
+        ],
+        "description": "Get a Threat Detection policy associated with an Azure SQL database.",
+        "operationId": "DatabaseThreatDetectionPolicies_Get",
+        "x-ms-examples": {
+          "Get database security alert policy": { "$ref": "../examples/DatabaseSecurityAlertGet.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "name": "serverName",
```

Comment:
Just for reducing duplicate declaration, you can declare serverName similar to ResourceGroupParameter, same is true to DatabaseName, specially if these are used a lot.